api-report/tree2.api.md

```diff
@@ -1578,6 +1573,45 @@ export class SharedTreeFactory implements IChannelFactory {
     type: string;
 }
 
+// @alpha
+export class SharedTreeView implements ISharedTreeView {
+    // (undocumented)
```

Comment:
That's always bothered me too. This is how API extractor has behaved for as long as I can remember. I know in the past we've used the "@inheritdocs" tag, and maybe API extractor respects that? But that's annoying to have to do considering that other tooling (e.g. VSCode mouse hover) understands just fine that a method with the same signature as the implemented interface has the same docs, by default.

Reply:
I filed an issue with API-Extractor about this a couple years ago, and has had 0 responses so far (since I last checked, anyways). This drives me crazy too. Fixing this might be a good hackathon project (he says, a week too late)...

Reply:
This reminds me that we had another api-extractor related project idea :)

Reply:
How about adding an "auto-generate" feature where any "undocumented" APIs will just have their docs generated by GPT?

Reply:
> This reminds me that we had another api-extractor related project idea :)

We did! But I was AFR OCE during FHL this time 😭